src/blog/2022/10/ff-docker-gcp.md

```diff
@@ -0,0 +1,180 @@
+- HTTPS access to the install
+- FlowForge [Device](https://flowforge.com/docs/user/concepts/#device) deployment via the included MQTT server that comes in our Docker build
+
+We will follow up with a second article covering the process of getting HTTPS running then we will close out the series by covering how to use key features of FlowForge including [Devices](https://flowforge.com/docs/user/concepts/#device).
+
+# Prerequisites
+
+- A domain name - We've registered flowforge-demo.com to demonstrate these steps
+- A DNS provider - Our Domain registrar provides a basic DNS service for free
+- A GCP account - Google will often give you free service credits on sign up so setting up FlowForge on GCP should not cost you anything for at least a few weeks
+- An email provider which will allow SMTP connections to send email - To manage users on your FlowForge platform you will need to be able to send emails to them. We have used a Google Workspace account for this purpose
+
+# GCP VM Creation
+
+Create a GCP account, once logged in navigate to Compute Engine then VM Instances. Select Create Instance you should now be [here](https://www.google.com/url?q=https://console.cloud.google.com/compute/instancesAdd?project).
+
+Give your instance a name, select a Region and Zone. I have found that the default Machine configuration works fine but depending on your project you may wish to change the resources.
+
+![GCP Create VM interface](images/1.png)
+
+You now need to allow access to your FlowForge installation from the internet. In the Firewall section tick Allow HTTP traffic and Allow HTTPS traffic.
+
+![GCP Create VM interface](images/2.png)
+
+You will now need to assign a static external IP address to this VM. Click Advanced options, then Networking. Now scroll down until you see Network interfaces and click on default to expand that section. In External IPv4 address select Create IP Address, give it a name then press Reserve.
+
+![GCP Create VM interface](images/3.png)
+
+Once you have reserved your IP it will be shown in the External IPv4 address field, write it down as we will need it later to create the DNS records. Our IP address was 34.125.156.130.
+
+![GCP Create VM interface](images/4.png)
+
+You are now ready to create and boot your VM, scroll to the bottom of the page and press Create. It can take a minute or two for the VM to be ready to use.
+
+# DNS Set Up
+
+So that you can run FlowForge on your newly created GCP VM you will need to set up 2 DNS records. These records are slightly different to what is suggested in the FlowForge install docs. We were keen to be able to run other services on this domain so we set up the following records.
+
+![DNS interface](images/5.png)
+
+Depending on your DNS provider it can take seconds to hours for these records to become live. Our’s were in place very quickly, You can validate the DNS records on a Mac or Linux PC using the dig command.
+
+![Dig command](images/6.png)
+
+The DNS we need is now in place.
```

Comment:
```suggestion
The DNS records are set to the IP record we noted down earlier, so we're good to continue.
```

Hope I'm not a nitpick, but felt like explaining in words too why it's good to go